PROTOCOL_V6_DRAFT.md

```diff
@@ -0,0 +1,201 @@
+# EXPERIMENTAL Twirp v6 Wire Protocol
+
+This document is a draft of a new version ("v6") of the Twirp wire
+protocol over HTTP. This draft is EXPERIMENTAL and subject to change.
+v6 of Twirp is not released or considered stable; instead, this
+document lists planned changes which may be available in prerelease
+distributions.
+
+## Changes from v5
+
+### URL scheme
+
+In [v5](./PROTOCOL.md), URLs followed this format:
+
+**URL ::= Base-URL "/twirp/" [ Package "." ] Service "/" Method**
+
+In v6, the "/twirp/" prefix is removed:
+
+**URL ::= Base-URL "/" [ Package "." ] Service "/" Method**
+
+The "/twirp/" prefix is removed for three reasons:
```

Comment:
The "/twirp/" prefix is not really removed. It can be part of the Base-URL if the service provider wants to keep it. It is just no longer part of the spec.

Reply:
Right, we should definitely allow path prefixes.